src/aws/lambda.tf

```diff
@@ -196,7 +196,6 @@ resource "aws_lambda_function" "marsha_mediapackage_lambda" {
       DISABLE_SSL_VALIDATION = var.update_state_disable_ssl_validation
       ECS_CLUSTER = aws_ecs_cluster.marsha_ffmpeg_transmux_cluster.arn
       ECS_TASK_DEFINITION = aws_ecs_task_definition.marsha_ffmpeg_transmux_definition.arn
-      ENDPOINT = "${var.marsha_base_url}${var.update_state_endpoint}"
```

Comment:
don't forget to add the `MARSHA_URL` environment variable

```
MARSHA_URL = var.marsha_base_url
```

Reply:
Oh, good catch, thanks